.github/workflows/build.yml

```diff
@@ -1,12 +1,12 @@
-name: CI Sidebar SDK JS
+name: CI Sidebar SDK JS v1.x.x
 
 on:
   push:
-    branches: [main]
+    branches: [version-1.x.x]
```

Comment:
Is this documented? If not then please document it. Confluence page will also do.